### PR TITLE
Update spring-boot version to 2.2 and spring-cloud to Hoxton

### PR DESCRIPTION
**Pull request by juliensadaoui**
related to https://github.com/jhipster/generator-jhipster/issues/10565

- Please make sure the below checklist is followed for Pull Requests.

- [ ] [Travis tests](https://travis-ci.org/jhipster/jhipster-registry/pull_requests) are green
- [ ] Tests are added where necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/jhipster-registry/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by juliensadaoui** [comment]
thanks a lot for update all dependencies @DanielFran !

**Comment by DanielFran** [comment]
@juliensadaoui Just missing update on maven-compiler plugin due to Java 11 and should be OK!

**Comment by juliensadaoui** [comment]
Spring boot 2.2.3 is release important which contains fixes for [CVE-2020-5398](https://pivotal.io/security/cve-2020-5398) and [CVE-2020-5397](https://pivotal.io/security/cve-2020-5397).

File: src/main/webapp/app/core/core.module.ts [hunk 1]
```diff
@@ -64,7 +63,6 @@ import { fontAwesomeIcons } from './icons/font-awesome-icons';
 export class JHipsterRegistryCoreModule {
   constructor(iconLibrary: FaIconLibrary, dpConfig: NgbDatepickerConfig) {
     registerLocaleData(locale);
-    iconLibrary.addIconPacks(fas);
```
Comment (juliensadaoui) [inline src/main/webapp/app/core/core.module.ts]: @DanielFran I have tested the registry with microservices connected, some icons doesn't show in the navbar

Comment (DanielFran) [reply src/main/webapp/app/core/core.module.ts]: This is related with https://github.com/jhipster/generator-jhipster/pull/11079 from @ruddell .
Will rollback that changes...

Comment (juliensadaoui) [reply src/main/webapp/app/core/core.module.ts]: I fixed the missing fontawesome icons, they must be imported in the font-awesome-icons.ts file

**Comment by juliensadaoui** [comment]
@DanielFran The Java 11 build failed, i don't understand. The VM settings is correct

```Preparing to unpack .../adoptopenjdk-11-hotspot_11.0.6+10-2_amd64.deb ...
Unpacking adoptopenjdk-11-hotspot (11.0.6+10-2) ...
Setting up adoptopenjdk-11-hotspot (11.0.6+10-2) ...
openjdk version "11.0.6" 2020-01-14
OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.6+10)
OpenJDK 64-Bit Server VM AdoptOpenJDK (build 11.0.6+10, mixed mode)
```

**Comment by DanielFran** [comment]
@juliensadaoui I know, I talk with @pascalgrimaud about this issue.

I found those 3 PR related with Java 11 migration:
- https://github.com/jhipster/generator-jhipster/pull/9539/files
- https://github.com/jhipster/generator-jhipster/pull/9799/files
- https://github.com/jhipster/generator-jhipster/pull/9841/files

The last 2 are related with this issue.

**Comment by juliensadaoui** [comment]
The Azure pipeline is green. The pipeline step starts a bash shell with `--noprofile` and `--norc` flags. The shell uses the default jvm inside the virtual machine. Below the output of `mvn -version` command:

```
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
Maven home: /home/vsts/.m2/wrapper/dists/apache-maven-3.6.3-bin/1iopthnavndlasol9gbrbg6bf2/apache-maven-3.6.3
Java version: 1.8.0_232, vendor: Azul Systems, Inc., runtime: /usr/lib/jvm/zulu-8-azure-amd64/jre
Default locale: en, platform encoding: UTF-8
OS name: "linux", version: "5.0.0-1027-azure", arch: "amd64", family: "unix"
```

I configured the jvm version (AdoptOpenJDK 11) in the step which builds/tests the server-side

```
sudo update-java-alternatives -s adoptopenjdk-11-hotspot-amd64
JAVA_HOME=$(readlink -f /usr/bin/java | sed "s:bin/java::")
./mvnw -version
./mvnw -ntp verify
```

**Comment by DanielFran** [comment]
@juliensadaoui Not sure maven.compiler.release is needed to be declared...

**Comment by juliensadaoui** [comment]
@DanielFran The `release` configuration is a new compiler parameter to replace the `source` and `target` version parameters (since Java 9)

**Comment by juliensadaoui** [comment]
You are right ! The `java.version` property is used for the `maven-compiler-plugin` configuration. The `maven.compiler.target` and `maven.compiler.release` are never used in the pom.xml

**Comment by DanielFran** [comment]
@pascalgrimaud @juliensadaoui Just waiting for the new version of jhipster bom and this is ready to be merged.

**Comment by DanielFran** [comment]
@pascalgrimaud Re-launched the tests!

**Comment by juliensadaoui** [comment]
I have tested the registry with microservices connected to it, and using jwt, oauth2 and uaa profiles. It works fine.

**Pull request merged**